describealign.py

```diff
@@ -28,13 +28,15 @@
 # nuitka-project-if: {OS} != "Windows":
 #    nuitka-project: --enable-plugins=pyside2
 #
+# nuitka-project: --standalone
 # Compilation mode, standalone everywhere, except on macOS there app bundle
 # nuitka-project-if: {OS} == "Darwin":
-#    nuitka-project: --standalone
 #    nuitka-project: --macos-create-app-bundle
 # Mac needs onefile too apparently, because pyside2 plugin requires it.
-# All other platforms need it to, so set it universally.
-# nuitka-project: --onefile
+# All other platforms need it too, so set it universally.

```

Comment:
Given this discovery, I feel like the default should be swapped and the if statement should convert for the single exception.  Bit of a tradeoff between readability and consistency with the repo's history

Reply:
What do you mean? like enable onefile by default and disable it for windows? I don't think we have an option to disable onefile, only to enable it

Reply:
Oops, brain fart had me thinking onefile and standalone were opposites